README.md

```diff
@@ -42,13 +44,11 @@ To run the project locally do the following:
   `cd School-library`
 - run the command:</br>
   `ruby main.rb`</br>
-- OR simply type</br>
-  `./main.rb`
 
 ## Built With
 
 - Major languages: Ruby
-- Tested with: - [Rubocop](https://rubocop.org/)
+- Tested with: - [Rubocop](https://rubocop.org/) and [Rspec](https://rspec.info/)
```

Comment:
- [ ] Let's update the readme file to include a command used for testing. Since this is a testing project, a user may want to know how to run tests on their own machine.